src/keygen/participant.rs

```diff
@@ -747,6 +751,9 @@ mod tests {
             let inbox = inboxes.get_mut(&participant.id).unwrap();
             inbox.push(participant.initialize_keygen_message(keyshare_identifier));
         }
+        let mut participant_ids = quorum[0].all_participants();
+        participant_ids.sort();
```

Comment:
question: does this need to be used somewhere?

Reply:
nop. good catch.